packages/gestalt/src/SegmentedControl/SegmentedControl.js

```diff
@@ -6,7 +6,7 @@ import Text from '../Text/Text';
 import styles from './SegmentedControl.css';
 
 type Props = {|
-  items: Array<string>,
+  items: Array<string | React.Node>,
```

Comment:
`React.Node` already covers `string`, you don't need to duplicate it
https://flow.org/en/docs/react/types/#toc-react-node

Reply:
@loserford We should also update the docs

Reply:
updated. @pfarejowicz @christianvuerings